packages/authentication/modules/access/useSignUp/index.ts

```diff
@@ -8,19 +8,26 @@ import { type SubmitHandler, useForm } from 'react-hook-form'
 
 import AuthApi from '../../../services/auth'
 import type { RegisterRequest } from '../../../types/auth'
-import { DEFAULT_INITIAL_VALUES, DEFAULT_VALIDATION_SCHEMA } from './constants'
+import {
+  DEFAULT_INITIAL_VALUES,
+  DEFAULT_VALIDATION_SCHEMA,
+  DEFAULT_VALIDATION_SCHEMA_WITH_NAME,
+} from './constants'
 import type { UseSignUpOptions } from './types'
 
 const useSignUp = <TRegisterRequest extends RegisterRequest, TRegisterResponse = void>({
   formOptions = {},
   ApiClass = AuthApi,
   enableFormApiErrors = true,
   options = {},
+  useNameField = false,
```

Comment:
shouldn't name be the default?

Reply:
We agreed to keep first and last name as default